Parser panics when double semi-colon in service option

## test case

Here is a simple reprod:

```proto
syntax = "proto3";
import "google/protobuf/descriptor.proto";

extend google.protobuf.ServiceOptions  {
  string service_description = 51000;
}

service MyService{
  option(service_description) = "description";;
}
```
note that there is a mistake in the file, with a double semicolon after description.

## actual behaviour

when running protolint on this file, we got a crash:


```

protolint lint test.proto
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x18 pc=0x527104]

goroutine 1 [running]:
github.com/yoheimuta/go-protoparser/v4/parser.(*Service).Accept(0xc0001b45a0, {0xb52a20, 0xc000008bd0})
        /home/runner/go/pkg/mod/github.com/yoheimuta/go-protoparser/v4@v4.7.0/parser/service.go:90 +0x64
github.com/yoheimuta/go-protoparser/v4/parser.(*Proto).Accept(0xc000222750?, {0xb52a20, 0xc000008bd0})
        /home/runner/go/pkg/mod/github.com/yoheimuta/go-protoparser/v4@v4.7.0/parser/proto.go:26 +0x69
github.com/yoheimuta/protolint/linter/visitor.RunVisitorAutoDisable({0xb53b88?, 0xc000224a80?}, 0xc000222750, {0xa7185b, 0x10}, 0x5?)
        /home/runner/work/protolint/protolint/linter/visitor/hasExtendedVisitor.go:55 +0x119
github.com/yoheimuta/protolint/linter/visitor.RunVisitor(...)
        /home/runner/work/protolint/protolint/linter/visitor/hasExtendedVisitor.go:28
github.com/yoheimuta/protolint/internal/addon/rules.QuoteConsistentRule.Apply({{{0xa68c01?, 0x15?}}, 0xc00001e82d?, 0xa?}, 0xc00019e600?)
        /home/runner/work/protolint/protolint/internal/addon/rules/quoteConsistentRule.go:60 +0xa7
github.com/yoheimuta/protolint/internal/linter.(*Linter).Run(0xc00000b6e0?, 0xc000177758, {0xc000228180?, 0x7, 0x0?})
        /home/runner/work/protolint/protolint/internal/linter/linter.go:33 +0x10e
github.com/yoheimuta/protolint/internal/cmd/subcmds/lint.(*CmdLint).runOneFile(0xc0001c4380, {{0xc00001e810, 0x27}, {0xc00001e82d, 0xa}})
        /home/runner/work/protolint/protolint/internal/cmd/subcmds/lint/cmdLint.go:131 +0x151
github.com/yoheimuta/protolint/internal/cmd/subcmds/lint.(*CmdLint).run(0xc0001c4380)
        /home/runner/work/protolint/protolint/internal/cmd/subcmds/lint/cmdLint.go:100 +0xd6
github.com/yoheimuta/protolint/internal/cmd/subcmds/lint.(*CmdLint).Run(0xc0001c4380)
        /home/runner/work/protolint/protolint/internal/cmd/subcmds/lint/cmdLint.go:77 +0x53
github.com/yoheimuta/protolint/internal/cmd.doLint({0xc0000360e0?, 0x100000000?, 0x0?}, {0xb46980, 0xc00005e028}, {0xb46980, 0xc00005e030})
        /home/runner/work/protolint/protolint/internal/cmd/cmd.go:110 +0x2d1
github.com/yoheimuta/protolint/internal/cmd.doSub({0xc0000360d0?, 0x0?, 0x1d8e0249df8?}, {0xb46980?, 0xc00005e028?}, {0xb46980?, 0xc00005e030?})
        /home/runner/work/protolint/protolint/internal/cmd/cmd.go:64 +0xac
github.com/yoheimuta/protolint/internal/cmd.Do({0xc0000360d0?, 0x60?, 0x0?}, {0xb46980?, 0xc00005e028?}, {0xb46980?, 0xc00005e030?})
        /home/runner/work/protolint/protolint/internal/cmd/cmd.go:49 +0x78
main.main()
        /home/runner/work/protolint/protolint/cmd/protolint/main.go:11 +0x5e
```

protoc correctly handles this by reporting a syntax error.

## Expected behaviour:
protolint should not panic, and rather output a syntax error like protoc

--- a/_testdata/double_semicolon_issue84/main.go
+++ b/_testdata/double_semicolon_issue84/main.go
@@ -1,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/yoheimuta/go-protoparser/v4/lexer"
+	"github.com/yoheimuta/go-protoparser/v4/parser"
+)
+
+// SimpleVisitor is a simple implementation of the parser.Visitor interface.
+type SimpleVisitor struct{}
+
+func (v *SimpleVisitor) VisitComment(*parser.Comment) {}
+func (v *SimpleVisitor) VisitDeclaration(*parser.Declaration) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitEdition(*parser.Edition) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitEmptyStatement(*parser.EmptyStatement) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitEnum(*parser.Enum) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitEnumField(*parser.EnumField) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitExtend(*parser.Extend) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitExtensions(*parser.Extensions) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitField(*parser.Field) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitGroupField(*parser.GroupField) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitImport(*parser.Import) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitMapField(*parser.MapField) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitMessage(*parser.Message) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitOneof(*parser.Oneof) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitOneofField(*parser.OneofField) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitOption(*parser.Option) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitPackage(*parser.Package) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitReserved(*parser.Reserved) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitRPC(*parser.RPC) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitService(*parser.Service) bool {
+	return true
+}
+func (v *SimpleVisitor) VisitSyntax(*parser.Syntax) bool {
+	return true
+}
+
+func main() {
+	// Proto file with double semicolon in service option
+	input := `
+syntax = "proto3";
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.ServiceOptions  {
+  string service_description = 51000;
+}
+
+service MyService{
+  option(service_description) = "description";;
+}
+`
+
+	// Parse the proto file
+	p := parser.NewParser(lexer.NewLexer(strings.NewReader(input)))
+	proto, err := p.ParseProto()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to parse proto: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Use the visitor pattern to process the parsed proto
+	visitor := &SimpleVisitor{}
+	proto.Accept(visitor)
+
+	fmt.Println("Successfully processed proto with double semicolon")
+}

--- a/parser/service.go
+++ b/parser/service.go
@@ -201,6 +201,7 @@ func (p *Parser) parseServiceBody() (
 			if err != nil {
 				return nil, nil, scanner.Position{}, err
 			}
+			stmt = &EmptyStatement{}
 		}
 
 		p.MaybeScanInlineComment(stmt)

--- a/parser/service_test.go
+++ b/parser/service_test.go
@@ -1109,6 +1109,49 @@ service SearchService {
 				},
 			},
 		},
+		{
+			name: "parsing a service with double semicolon in option",
+			input: `
+service MyService {
+  option(service_description) = "description";;
+}
+`,
+			wantErr: false,
+			wantService: &parser.Service{
+				ServiceName: "MyService",
+				ServiceBody: []parser.Visitee{
+					&parser.Option{
+						OptionName: "(service_description)",
+						Constant:   `"description"`,
+						Meta: meta.Meta{
+							Pos: meta.Position{
+								Offset: 23,
+								Line:   3,
+								Column: 3,
+							},
+							LastPos: meta.Position{
+								Offset: 66,
+								Line:   3,
+								Column: 46,
+							},
+						},
+					},
+					&parser.EmptyStatement{},
+				},
+				Meta: meta.Meta{
+					Pos: meta.Position{
+						Offset: 1,
+						Line:   2,
+						Column: 1,
+					},
+					LastPos: meta.Position{
+						Offset: 69,
+						Line:   4,
+						Column: 1,
+					},
+				},
+			},
+		},
 	}
 
 	for _, test := range tests {

